onnxruntime/core/platform/windows/env_time.cc

```diff
@@ -28,9 +28,12 @@ namespace {
 
 class WindowsEnvTime : public EnvTime {
  public:
+#if WINVER >= _WIN32_WINNT_WIN8
+  WindowsEnvTime() : GetSystemTimePreciseAsFileTime_(GetSystemTimePreciseAsFileTime) {}
+#else
   WindowsEnvTime() : GetSystemTimePreciseAsFileTime_(NULL) {
-    // GetSystemTimePreciseAsFileTime function is only available in the latest
-    // versions of Windows. For that reason, we try to look it up in
+    // GetSystemTimePreciseAsFileTime function is only available in
+    // Windows >= 8. For that reason, we try to look it up in
```

Comment:
>Windows >= 8. [](start = 7, length = 13)

Do we need win7 support here? My guess is this is related to wanting telemetry for the edge builds?

Reply:
I thought we wanted Win7 support in ORT/WinML?